src/components/Checkout.vue

```diff
@@ -131,7 +139,11 @@ export default {
         .then((response) => {
           if (response.status === 200) {
             this.cartItems = [...response.data];
-            this.buyer = JSON.parse(localStorage.getItem("personalData") || {});
+            const buyerRecord = JSON.parse(
+              localStorage.getItem("personalData") || {}
+            );
```

Comment:
應該是這樣才對？

```suggestion
            const buyerRecord = JSON.parse(localStorage.getItem("personalData")) || {};
```

因為 `JSON.parse({})` 會壞掉

Reply:
改好了~　https://github.com/c711cat/lemon_coffee/pull/42/commits/6d75318a0968df64974228fbb6bce633c612a991